data/map systems.txt

```diff
@@ -11685,6 +12304,68 @@ system "Elabi Choati"
 		distance 2183.75
 		period 745.253
 
+system Electra
+	pos -275.77778 73.111113
+	government Syndicate
+	link Botein
+	link Iklil
+	link Puppis
+	asteroids "small rock" 9 3.4362
+	asteroids "medium rock" 8 3.4362
+	asteroids "large rock" 3 2.07
+	asteroids "small metal" 5 2.898
+	asteroids "medium metal" 6 2.4426
+	asteroids "large metal" 3 2.1114
+	minables silver 7 4.3545762
+	minables copper 8 5.7354391
+	haze _menu/haze-133
+	fleet "Large Core Pirates" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
```

Comment:
```suggestion
	fleet "Small Core Pirates" 1000
	fleet "Large Core Pirates" 2500
	fleet "Small Core Merchants" 1800
```
This is away from any main routes, so the main activity here is going to be pirates stalking lost merchants.